CONTRIBUTING.md

```diff
@@ -34,15 +34,11 @@ If you don't have npm, it comes with node. To get it via homebrew:
 `brew install node`
 
 To add or update a dependency:
-- `go get -u` to update the dependencies or `go get {package}` to add a dependency
+- `cat GLOCKFILE | grep -oE '[^[:space:]]+\.[^[:space:]]+' | sort | uniq | xargs go get -u` to update the dependencies or `go get {package}` to add a dependency
```

Comment:
Oh, I missed that. Yes, that would work, but that's basically the nuclear option and sucks big time for people who work on other go projects.